styles/styles-sheet.css

```
@@ -0,0 +1,201 @@
+body  {
+    background-image: url(https://wallpapercave.com/wp/wp4047726.jpg);
+    background-size: auto;
+    /* background-color: #cccccc; */
+  }
+
+
+
+/* header {
+background-color: rgb(0, 0, 0);
+height: 100px;
+font-family: Arial, Helvetica, sans-serif;
+} */
+
+
+.nav-wrapper {
+  display: flex;
+  justify-content: space-between;
+  padding: 38px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.left-side {
+    display: flex;
+}
+
+.nav-wrapper > .left-side > section {
+    /* margin-right: 20px; */
+    margin: auto;
+    margin-left: 90%;
+    font-size: 0.9rem;
+    text-transform: uppercase;
+}
+
+.nav-link-wrapper {
+    height: 22px;
+    border-bottom: 1px solid transparent;
+    transition: border-bottom 0.5s;
+}
+
+.nav-link-wrapper a {
+    color:white;
+    text-decoration: none;
+    transition: color 0.5s;
+}
+
+.nav-link-wrapper:hover {
+    border-bottom: 1px solid whitesmoke;
+}
+
+.nav-link-wrapper a:hover {
+    color: whitesmoke;
+}
+
+a {
+    color: #ffffff;
+    display:inline-block;
+    margin-top: 50px;
+}
+
+
+.title-name {
+    color: #ffffff;;
+    font-size: 100px;
+    font-family: Arial, Helvetica, sans-serif;
+    font-weight:bold;
+    font-kerning: normal;
+    margin-top: 500%;
+    margin: auto;
+    text-align: center;
+    text-shadow:2px 2px #818181;
+    padding-left: 70px;
+    padding-top: 10%;
+    
+}
+
+.title-description {
+    color: #858585;;
+    /* font-size: 100px; */
+    font-family: Arial, Helvetica, sans-serif;
+    font-weight: lighter;
+    font-kerning: normal;
+    /* margin-bottom: 0px;
+    margin-top: 200px; */
+    text-align: center;
+    /* text-shadow:1px 1px #cfcfcf; */
+    padding-left: 50px;
+    
+
+}
+
+.iframe-container-about {
+    width: 560px; /* you have to have a size or this method doesn't work */
+    height: 315px; /* think about making these max-width instead - might give you some more responsiveness */
+
+    /* position: absolute; positions out of the flow, but according to the nearest parent */
+    top: 7; right: 0; /* confuse it i guess */
```

You need to specify a unit here.

```suggestion
    top: 7px; right: 0px; /* confuse it i guess */
```